docs/index.rst

```diff
@@ -199,6 +199,25 @@ In this example we use the python json tool to pretty print the json output:
       }
   }
 
+Configuration file
```

Comment:
This is great. Perhaps you could write something here about that the location of the default config file can be seen when running the program, so that people who wonder where it is located can find it?